server/src/resolvers/StructureDefinition.ts

```diff
@@ -61,12 +62,23 @@ export const StructureDefinition = objectType({
       },
     })
 
-    t.field('attributes', {
-      type: 'JSON',
+    t.list.field('attributes', {
+      type: 'AttributeDefinition',
       description: 'Structured version of the attributes',
       resolve: (parent: any) => parent.attributes,
     })
 
+    t.list.field('extensions', {
+      type: 'StructureDefinition',
+      description: 'List of extensions on this type',
```

Comment:
Maybe it would be clearer to say 'List of allowed/possible extensions on this type.
And I think I'm a bit lost here because I thought the extension depended on the context (which is ~ a path)

Reply:
fixed the description. In the context of a StructureDefinition, we don't have any context information (where is the StructureDefinition defined in a fhir object) so we only fetch the extensions that apply on this StructureDefinition in any context. However, when dealing with AttributeDefinition, we DO have context (in the cached attribute definition, the id provides information eg: `Observation.category:VSCat.text`) so we can also fetch all extensions that apply in this specific context

Reply:
Hum, right, I thought that maybe we could have extensions with a context that is not directly in a single structure definition but it seems that it won't be the case 👍